identity/webapp/src/router.ts

```diff
@@ -38,18 +39,20 @@ export const router = new TypedRouter({
   logout: [TypedRouter.GET, '/logout', logoutAction],
   callback: [TypedRouter.GET, '/callback', authCallback],
 
-  // Proxy apis.
+  // Proxy apis - implemented
+  'create-user': [TypedRouter.POST, '/api/user/create', registerUser, 'RegisterUserSchema'],
+  'get-user': [TypedRouter.GET, '/api/users/:user_id', getUser],
```

Comment:
We're expecting the front end to query `/api/users/:user_id` here, but the front end doesn't know the `user_id`.

We also don't need the `user_id` to be part of the request here, as we can get it from the session.

Should we change this endpoint's URL? Something like `/api/users/me` or `/api/users/current`?

Reply:
Yeah, I think something like `/api/users/me` is probably the right call here - there's no need to make the UI aware of what the user's ID is just for the sake of making the proxy API match the remote API exactly IMO

Reply:
Thinking more on this, we're right, but we'd need to leave the door open for an Admin to view a single user by their ID. So I've added a new `/api/users/me` endpoint here, and left `/api/users/:user_id` as it was before this PR.

Reply:
I've been working on the assumption that the admin app was a separate application in this same repo, which means we don't need to worry about admin users here. We'll call the same remote API from both applications but the two applications don't need the same proxy APIs.